src/DynamoCore/DSEngine/EngineController.cs

```diff
@@ -43,7 +40,7 @@ private void OnTraceReconciliationComplete(TraceReconciliationEventArgs e)
             }
         }
 
-        private readonly LiveRunnerServices liveRunnerServices;
+        public readonly LiveRunnerServices liveRunnerServices;
```

Comment:
Thanks for catching it.

On Tue, May 12, 2015, 6:50 PM Peter Boyer <peter.b.boyer@gmail.com> wrote:

> This is just a mistake.
>
> On Tue, May 12, 2015, 10:12 AM Luke Church <notifications@github.com>
> wrote:
>
>> In src/DynamoCore/DSEngine/EngineController.cs
>> <https://github.com/DynamoDS/Dynamo/pull/4424#discussion_r30137192>:
>>
>> > @@ -43,7 +40,7 @@ private void OnTraceReconciliationComplete(TraceReconciliationEventArgs e)
>> >              }
>> >          }
>> >
>> > -        private readonly LiveRunnerServices liveRunnerServices;
>> > +        public readonly LiveRunnerServices liveRunnerServices;
>>
>> Why do we need to make this public, is this for future work? I can't find
>> references to it in this PR, and the naming doesn't follow the right scheme
>> for a public API.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/DynamoDS/Dynamo/pull/4424/files#r30137192>.
>>
>